chaos/table_chaos_get_retry_no_config.go

```diff
@@ -0,0 +1,58 @@
+package chaos
+
+import (
+	"context"
+	"errors"
+	log "log"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+)
+
+func getRetryNoConfigTable() *plugin.Table {
+	return &plugin.Table{
+		Name:        "chaos_get_retry_no_config",
+		Description: "Chaos table to test the Get function with Default Retry config defined at plugin level in case of non fatal error",
+		List: &plugin.ListConfig{
+			Hydrate: getRetryNoConfigList,
+		},
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("id"),
+			Hydrate:    getRetryNoConfig,
+		},
+		Columns: []*plugin.Column{
+			{Name: "id", Type: proto.ColumnType_INT},
+			{Name: "retriable_error", Type: proto.ColumnType_STRING},
+		},
+	}
+}
+
+func getRetryNoConfigList(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	log.Println("[INFO] INSIDE LIST CALL")
```

Comment:
do we need to keep these logs? If you feel they are useful, let's make them lower case

Reply:
also, make them DEBUG level
```
"[DEBUG] Inside chaos_get_retry_no_config List call")
```